src/components/assets/Asset.js

```diff
@@ -9,23 +9,42 @@ function Asset ({asset}) {
   return <div styleName='asset-container'>
     <div styleName='preview'>
       <a href={`${asset.fields.file.url}`} target='_blank'>
-        <Thumbnail url={asset.fields.file.url} fileName={asset.fields.file.fileName} size='320' />
+        <Thumbnail url={asset.fields.file.url} fileName={asset.fields.file.fileName} size='500' />
       </a>
     </div>
     <div styleName='metadata'>
-      <h1>{asset.fields.title}</h1>
-      <h2>Description</h2>
-      <p>{asset.fields.description}</p>
-      <h2>Creation Date</h2>
-      <p>{formatDate(asset.sys.createdAt)}</p>
-      <h2>File Name</h2>
-      <p>{asset.fields.file.fileName}</p>
-      <h2>URL</h2>
-      <p><a href={`${asset.fields.file.url}`} target='_blank'>http:{asset.fields.file.url}</a></p>
-      <h2>MIME Type</h2>
-      <p>{asset.fields.file.contentType}</p>
-      <h2>Size</h2>
-      <p>{prettyBytes(asset.fields.file.details.size)}</p>
+      <section styleName='left'>
+        <div>
+          <h3>Title</h3>
+          <h2>{asset.fields.title}</h2>
+        </div>
+        <div>
+          <h3>Description</h3>
+          <p>{asset.fields.description || 'No description provided.'}</p>
+        </div>
+        <div>
+          <h3>Creation Date</h3>
+          <p>{formatDate(asset.sys.createdAt)}</p>
+        </div>
+      </section>
+      <section>
+        <div>
+          <h3>File Name</h3>
+          <p>{asset.fields.file.fileName}</p>
+        </div>
+        <div>
+          <h3>URL</h3>
+          <p><a href={`${asset.fields.file.url}`} target='_blank'>http:{asset.fields.file.url}</a></p>
```

Comment:
Not a big deal but we can always prefix these `hrefs` with `https:` by defaut, as we know we have all of our assets under https and that means we'll force https either way (and it's generally a good practice). In this particular case probably even more as this is user generated content.